colors/ascetic.vim

```diff
@@ -112,6 +112,12 @@ hi! link PreProc      Constant
 hi! link Type         Constant
 hi! link Special      Constant
 
+if s:ascetic_highlight_strings
+  call s:h("String",  {"fg": s:col.medium_gray})
```

Comment:
I used the "raw" medium gray because I was not getting consistently good-looking results across dark/light schemes with the calculated color values.

Reply:
Any chance you can show me a screenshot of what this looks like with dark/light?

Reply:
Here you go!
![dark](https://github.com/user-attachments/assets/942d1353-cd6e-488f-aed3-2e53e907e4dd)
![light](https://github.com/user-attachments/assets/00648873-8a93-4409-a1d9-d24914a4d8c8)